### PR TITLE
Adjust python tests for windows

### PR DESCRIPTION
**Pull request by mzegla**
### 🛠 Summary

JIRA/Issue if applicable.
Describe the changes.

### 🧪 Checklist

- [ ] Unit tests added.
- [ ] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

File: src/test/pythonnode_test.cpp [hunk 1]
```diff
@@ -2511,6 +2626,9 @@ TEST_F(PythonFlowTest, FinalizeException) {
             }
         }
     )"};
+#ifdef _WIN32
+    replaceHandlerPath(pbTxt);
```
Comment (rasapala) [inline src/test/pythonnode_test.cpp:2630]: Add ifdef inside the replace function instead of in every occurance.

**Review by rasapala — CHANGES_REQUESTED** [review]
Move IFdef win32 inside the replaceHandlerPath function.

File: demos/python_demos/seq2seq_translation/README.md [hunk 2]
```diff
@@ -24,6 +26,53 @@ cd demos/python_demos/seq2seq_translation
 docker run -it --rm -p 9000:9000 -v ${PWD}/servable:/workspace openvino/model_server:py --config_path /workspace/config.json --port 9000
 ```
 
+## Running Model Server on Windows (experimental)
+
+### Build Model Server binary with Python enabled
+For the Model Server to include embedded Python interpreter you need to [build the binary from scratch](../../../docs/windows_developer_guide.md). Note the `--define MEDIAPIPE_DISABLE=0 --define PYTHON_DISABLE=0` part in `bazel build` command that specifies that both MediaPipe and Python support will be included (Python support implementation is based on MediaPipe).
```
Comment (rasapala) [inline demos/python_demos/seq2seq_translation/README.md:32]: Note the `--define MEDIAPIPE_DISABLE=0 --define PYTHON_DISABLE=0` part in `.bazelrc` file in # Windows config default flags section - build:windows,  that specifies that both MediaPipe and Python support will be included (Python support implementation is based on MediaPipe).

File: demos/python_demos/seq2seq_translation/README.md [hunk 3]
```diff
@@ -24,6 +26,53 @@ cd demos/python_demos/seq2seq_translation
...
+### Build Model Server binary with Python enabled
+For the Model Server to include embedded Python interpreter you need to [build the binary from scratch](../../../docs/windows_developer_guide.md). Note the `--define MEDIAPIPE_DISABLE=0 --define PYTHON_DISABLE=0` part in `bazel build` command that specifies that both MediaPipe and Python support will be included (Python support implementation is based on MediaPipe). 
+
+Next steps assume you have followed the building instruction from the developer guide. If you chose different download or installation paths you will need to adjust below instructions.
+
+### Prepare environment for Python servable deployment
+
+Once you have OVMS executable, there are three more steps for the Python servable to work.
+
+1. Create a copy of `pyovms.so` file created during the build with `.pyd` extension in the same location.
```
Comment (rasapala) [inline demos/python_demos/seq2seq_translation/README.md:40]: I think we need to automate this step in our windows build.

File: demos/python_demos/seq2seq_translation/README.md [hunk 4]
```diff
@@ -24,6 +26,53 @@ cd demos/python_demos/seq2seq_translation
...
+
+    Go to the location where Python binding building artifacts are stored:
+
+    `cd  C:\git\model_server\bazel-out\x64_windows-opt\bin\src\python\binding`
+
+    Then copy `pyovms.so` to `pyovms.pyd`:
+
+    `cp pyovms.so pyovms.pyd`
+
+2. Set environment variables for the `pyovms` and `openvino` bindings:
```
Comment (rasapala) [inline demos/python_demos/seq2seq_translation/README.md:50]: cmd.exe vs ps.exe

**Review by rasapala — COMMENTED** [review]
Comments inline.

File: demos/python_demos/seq2seq_translation/README.md (hunk 2 above)
Comment (mzegla) [reply demos/python_demos/seq2seq_translation/README.md]: Removed entirely

File: demos/python_demos/seq2seq_translation/README.md (hunk 3 above)
Comment (mzegla) [reply demos/python_demos/seq2seq_translation/README.md:40]: Let's do this in a separate PR, so we don't mix too many topics in this one

File: ci/loadWin.groovy [hunk 5]
```diff
@@ -23,18 +23,30 @@ def check_tests(){
     status = bat(returnStatus: true, script: 'grep "       OK " win_test.log')
     if (status != 0) {
             error "Error: Windows run test failed ${status}. Expecting passed tests and no passed tests detected. Check win_test.log for details."
+    } else {
+        def passed = bat(returnStatus: false, returnStdout: true, script: 'grep "       OK " win_test.log | wc -l')
+        echo "Error: Windows run test passed ${status}. ${passed} passed tests . Check win_test.log for details."
```
Comment (rasapala) [inline ci/loadWin.groovy:28]: echo "Success : Windows run test passed ${status}. ${passed} passed tests . Check win_test.log for details."

File: build_windows.bat [hunk 6]
```diff
@@ -22,16 +22,19 @@ set "bazelStartupCmd=--output_user_root=%BAZEL_SHORT_PATH%"
 
 set "buildCommand=bazel %bazelStartupCmd% build --config=windows --jobs=%NUMBER_OF_PROCESSORS% --verbose_failures //src:ovms 2>&1 | tee win_build.log"
 set "buildTestCommand=bazel %bazelStartupCmd% build --config=windows --jobs=%NUMBER_OF_PROCESSORS% --verbose_failures //src:ovms_test 2>&1 | tee win_build_test.log"
+set "copyPyovms=cp %cd%\bazel-out\x64_windows-opt\bin\src\python\binding\pyovms.so %cd%\bazel-out\x64_windows-opt\bin\src\python\binding/pyovms.pyd"
```
Comment (dkalinowski) [inline build_windows.bat:25]: are you sure this is correct? you are mixing `\` with `/`

Comment (mzegla) [reply build_windows.bat:25]:
```suggestion
set "copyPyovms=cp %cd%\bazel-out\x64_windows-opt\bin\src\python\binding\pyovms.so %cd%\bazel-out\x64_windows-opt\bin\src\python\binding\pyovms.pyd"
```

It works, but I can change it to be uniform

File: ci/loadWin.groovy (hunk 5 above)
Comment (mzegla) [reply ci/loadWin.groovy:28]:
```suggestion
        echo "Success: Windows run test passed ${status}. ${passed} passed tests . Check win_test.log for details."
```

File: src/test/streaming_test.cpp [hunk 7]
```diff
@@ -1060,6 +1096,9 @@ node {
     }
 }
 )"};
+#ifdef _WIN32
+    replaceHandlerPath(testPbtxt);
```
Comment (dkalinowski) [inline src/test/streaming_test.cpp:1100]: If we change replaceHandlerPath to return modified file:
```suggestion
    textPbtext = replaceHandlerPath(testPbtxt);
```

We can follow previous naming, like `getGenericFullPathForTmp`, for example:
// getGenericGraphPbtxt()

and then move if def inside it

**Review by rasapala — APPROVED** [review]
LGTM

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**